utils/appliance/__init__.py

```diff
@@ -249,37 +250,103 @@ def seal_for_templatizing(self):
             ssh_client.run_command('echo "vm.swappiness = 1" >> /etc/sysctl.conf',
                 ensure_host=True)
 
+    def _encrypt_string(self, string):
+        try:
+            # Let's not log passwords
+            logging.disable(logging.CRITICAL)
+            rc, out = self.ssh_client.run_rails_command(
+                "puts MiqPassword.encrypt('{}')".format(string))
+            return out
+        finally:
+            logging.disable(logging.NOTSET)
+
+    def _get_ems_ips(self, ems):
```

Comment:
As we are going to need to support IPv6 in the future, is that something we should start to consider here too?